docs/API.md

```diff
@@ -192,11 +259,11 @@ scope.activate(outerSpan, async () => {
   scope.bind(outerEmitter)
 
   innerEmitter.on('request', () => {
-    log(scope.active()) // innerSpan because explicitly bound
+    log(scope.active()) // innerSpan because it was explicitly bound
```

Comment:
Do you intend to change this to `console.log` for consistency (since you changed the others)?